packages/default-theme/cms/elements/CmsElementProductCard.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <SwProductCard
+    :product="content.data.product"
```

Comment:
we can also get the product object from computed property to avoid lot of dots here ;) or a helper is a good way to extract it? i don't know :+1:

Reply:
will be covered with issue https://github.com/DivanteLtd/shopware-pwa/issues/636